packages/ui-components/error-boundary/README.md

```diff
@@ -0,0 +1,15 @@
+# @k8slens/error-boundary
+
+This package contains stuff related to creating Lens-applications. 
```

Comment:
Copypaste? ;)

Reply:
naturally

Reply:
noop.  does not look like it is needed.  it is removed now